requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
I think this does not change anything, but we changed the pip-compile python version here.

Reply:
yes, it's probably because my default python is 3.8. I'll generate this with python 3.9

Reply:
done